tests/test_cli.py

```diff
@@ -115,3 +128,81 @@ def test_csv_generation():
                 assert expected_csv == fd.read()
         finally:
             shutil.rmtree(tempdir)
+
+
+@pytest.fixture(scope="function")
+def sqlite_store():
+    fd, temp_dbfile = tempfile.mkstemp()
+    # Close handle immediately so that we can remove the file later on in Windows
+    os.close(fd)
+    db_uri = "sqlite:///%s" % temp_dbfile
+    store = SqlAlchemyStore(db_uri, 'artifact_folder')
+    yield (store, db_uri)
+    os.remove(temp_dbfile)
+    shutil.rmtree('artifact_folder')
+
+
+@pytest.fixture(scope="function")
+def file_store():
+    ROOT_LOCATION = os.path.join(tempfile.gettempdir(), 'test_mlflow_gc')
+    file_store_uri = "file:///%s" % ROOT_LOCATION
+    yield (FileStore(ROOT_LOCATION), file_store_uri)
+    shutil.rmtree(ROOT_LOCATION)
+
+
+def _create_run_in_store(store):
+    config = {
+        'experiment_id': '0',
+        'user_id': 'Anderson',
+        'start_time': int(time.time()),
+        'tags': {}
+    }
+    run = store.create_run(**config)
+    artifact_path = url2pathname(unquote(urlparse(run.info.artifact_uri).path))
+    if not os.path.exists(artifact_path):
+        os.makedirs(artifact_path)
+    return run
+
+
+def test_mlflow_gc_sqlite(sqlite_store):
+    store = sqlite_store[0]
+    run = _create_run_in_store(store)
+    store.delete_run(run.info.run_uuid)
+    subprocess.check_output(["mlflow", "gc", "--backend-store-uri", sqlite_store[1]])
+    runs = store.search_runs(experiment_ids=['0'], filter_string='', run_view_type=ViewType.ALL)
+    assert len(runs) == 0
+    with pytest.raises(MlflowException):
+        store.get_run(run.info.run_uuid)
```

Comment:
Do you really need to check this ? I think trusting search_runs which is tested somewhere else is enough no ?

Reply:
You're right the check was not usefull, I removed it